model/org.eclipse.birt.report.model/src/org/eclipse/birt/report/model/api/DesignElementHandle.java

```diff
@@ -1608,7 +1600,7 @@ public void copyPropertyTo(String propName, DesignElementHandle targetHandle) th
 		case IPropertyType.LIST_TYPE:
 
 			assert value instanceof List;
-			List valueList = (List) value;
+			List<?> valueList = (List<?>) value;
```

Comment:
A for-each loop below would be simpler/clearer.

Reply:
What is the suggestion?

Reply:
```
			assert value instanceof List;
			List<?> valueList = (List<?>) value;
			PropertyHandle propHandle = targetHandle.getPropertyHandle(propName);
			for (Object item : valueList) {
```

Again, it's your option.

Reply:
Loop is changed.